inference-engine/tests/functional/plugin/shared/include/single_layer_tests/reduce_ops.hpp

```diff
@@ -8,19 +8,20 @@
 #include <string>
 #include <vector>
 #include <memory>
+#include <utility>
 
 #include "functional_test_utils/layer_test_utils.hpp"
 #include "ngraph_functions/builders.hpp"
 
 namespace LayerTestsDefinitions {
 
 typedef std::tuple<
-        std::vector<int>,               // Axis to reduce order
-        bool,                           // Keep dims
-        ngraph::helpers::ReductionType, // Reduce operation type
-        InferenceEngine::Precision,     // Net precision
-        std::vector<size_t>,            // Input shapes
-        std::string                     // Target device name
+        std::pair<std::vector<int>, size_t>,    // Axis to reduce order and shape size
```

Comment:
I don't think this is a good way for testing Scalar axis case. From my understanding it is better to have enum parameter which will determine axis input type: Scalar/Vector. Please see Eltwise single layer test for example.
@iefode What do you think about that? I would also say that we need to extend other test so the will cover 0d axis input as well. I think we may find some issues in runtime due to this coverage.

Reply:
extract enum opType to [common_layers_params.hpp](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/ie_test_utils/common_test_utils/common_layers_params.hpp)
split test instance in two (0d 1d with 1 axis and 1d with number of axis not equal 1)